client/state/plugins/installed/actions.js

```diff
@@ -191,7 +191,7 @@ function installPluginHelper( siteId, plugin, isMainNetworkSite = false ) {
 		dispatch( { ...defaultAction, type: PLUGIN_INSTALL_REQUEST } );
 
 		const doInstall = function( pluginData ) {
-			return getPluginHandler( siteId, pluginData.id ).install();
+			return getPluginHandler( siteId, pluginData.slug ).install();
```

Comment:
Changing the behavior in the core action makes me nervous. What is the impact on the rest of Calypso? Is there anyway we could import what we need and wrap our own install action? At the very least I think this change would need reviewed independent of this PR by the Calypso team.